backend/src/utils/create-validator.ts

```diff
@@ -0,0 +1,5 @@
+import { JSONSchema } from 'json-schema-to-ts';
+import { getAjvClient } from './get-ajv-client';
+
+export const createValidator = (schema: JSONSchema) =>
```

Comment:
**Question**: Do we not want to feed in all our schemas into this one so the refs will work?

Reply:
We do indeed, I'll make it so.